Resources/Prototypes/Entities/Clothing/OuterClothing/sinditrench.yaml

```
@@ -0,0 +1,20 @@
+- type: entity
+  parent: ClothingOuterStorageBase
+  id: Republicssinditrench
+  name: офицерский тренч синдиката
```

```suggestion
  name: офицерский тренч синдиката
```
Надо через локализацию